spec/enumerable_spec.rb

```diff
@@ -0,0 +1,197 @@
+require 'spec_helper'
+require './lib/enumerable'
+
+RSpec.describe Enumerable do
+  describe '#my_each' do
+    it 'should return each array element as it is' do
```

Comment:
It's best practice to start the `it` description with a singular verb.
```suggestion
    it 'returns each array element as it is' do
```